easygui/boxes/text_box.py

```diff
@@ -209,8 +207,8 @@ def demo_textbox(reply):
         * 5) + "\n\n") * 10
     title = "Demo of textbox"
     msg = "Here is some sample text. " * 16
-    reply = textbox(msg, title, text_snippet)
+    reply = db.textbox(msg, title, text_snippet)
     ut.writeln("Reply was: {!s}".format(reply))
 
 if __name__ == '__main__':
-    print demo_textbox()
+    ut.writeln(demo_textbox())
```

Comment:
I'll make this update, but just changing this to print(demo_textbox()) is all that is needed.